packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -1275,6 +1304,18 @@ class _FlutterApiTestImplementation implements FlutterIntegrationCoreApi {
   @override
   void noop() {}
 
+  // TODO(tarrinneal): update these to send error to platform,
+  // See: https://github.com/flutter/flutter/issues/118243
```

Comment:
Do we need a TODO here? If we just add a bit more to the FlutterError so we can look for specific parts of it, the implementation should be good to go.